src/extension/background-script/state.ts

```diff
@@ -20,11 +22,15 @@ interface State {
   currentAccountId: string | null;
   nostrPrivateKey: string | null;
   nostr: Nostr | null;
+  mnemonic: Mnemonic | null;
```

Comment:
Add TODO to move these to the account level of state (then they can handle the decryption)